app/reducers/events.ts

```diff
@@ -192,7 +195,7 @@ export const selectUpcomingEvents = createSelector(selectEvents, (events) =>
   events.filter((event) => event.isUsersUpcoming)
 );
 export const selectSortedEvents = createSelector(selectEvents, (events) =>
-  events.sort((a, b) => a.startTime - b.startTime)
+  [...events].sort((a, b) => a.startTime.unix() - b.startTime.unix())
```

Comment:
What's the point of `[...events]`? Is `events` not already an array of `Event`?

Reply:
JS `.sort` is in-place. Both being annoying as mutating arrays when doing things like this is a bad idea IMO, and the fact that `events` already has a type, and the result of the `.sort()` is something else, and TS can't magically change the type of a variable. So creating a copy ensures that TS infers the correct type.

Reply:
ah yes that makes perfect sense, I forgot that it was in-place.